tests/unit/maxmemory.tcl

```diff
@@ -22,6 +22,18 @@ start_server {tags {"maxmemory" "external:skip"}} {
         assert_equal [r dbsize] 50
     }
     
+    proc check_test {client_eviction} {
```

Comment:
this function is VERY specific for the test that uses it, but it's name is very generic.
please rename it and also add some comment explaining what it does / used for.

let's also do that for `verify_test`